pkg/template/helper.go

```diff
@@ -42,3 +44,35 @@ func (r TemplateReference) String() string {
 	}
 	return r.Name
 }
+
+// ConvertUserToTemplateInstanceRequester copies analogous fields from user.Info to TemplateInstanceRequester
+func ConvertUserToTemplateInstanceRequester(u user.Info) templateapi.TemplateInstanceRequester {
```

Comment:
we cannot unless minimally the upstream orig hdr spec changes .... json marshalling/unmarshalling of DefaultInfo does not cut it with that